src/app/containers/Brand/index.test.jsx

```diff
@@ -30,3 +33,18 @@ describe(`BrandContainer`, () => {
     BrandContainerWithContext(newsServiceContextStub),
   );
 });
+
+describe('BrandContainer with variant', () => {
+  const service = 'serbian';
+  const variant = 'cyr';
+  shouldMatchSnapshot(
+    'should render correctly',
```

Comment:
what is different about a brandcontainer with a variant?

Reply:
With a variant it will show a ScriptLink. Doesn't have a ScriptLink without variant.

Reply:
OK that's not clear it's a requirement from the test because it's not in the title and outputs a snapshot instead of explicitly checking for a script link.

Reply:
Yes. This is the issue I mention in the description. `ScriptLink` isn't displaying in `Brand` component as required. @hindsc52 mentioned the frontpage pod will fix this integration.

Reply:
OK then can we remove this test and instead ask the front page pod to write a suitable test?